server/pbench/bin/gold/test-7.8.txt

```diff
@@ -1039,7 +1039,19 @@ len(actions) = 19
         },
         "_type": "pbench-tool-data-iostat"
     }
-]
+]could not convert binary: /var/tmp/pbench-test-server/tmp/uperf__2016-10-06_16:34:03/1-tcp_stream-16384B-32i/sample1/tools-default/flat8_40gb/sar/sar.data
```

Comment:
We need to resolve this issues.  Are you expecting to have test sar data made available in the unit test for this one?